Repaired spelling errors. Afects lintian warnings in freefoam and libfreefoam packages
--- a/doc/man/manSource/freefoam.1.in.txt
+++ b/doc/man/manSource/freefoam.1.in.txt
@@ -35,7 +35,7 @@
 
 - It searches in the directory '<libexecDir>' for an executable with the name
   '@FOAM_EXE_PREFIX@<name>'. '<libexecDir>' defaults to '@FOAM_INSTALL_LIBEXEC_PATH@'.
-  This can be overriden using the '-b' option.
+  This can be overridden using the '-b' option.
 - If the +FREEFOAM_PATH+ variable is set, it is prepended to the above
   described search path. It is a colon-separated list of directories.
 - The '-P' option can be used to append the system +PATH+ to the search path.
@@ -236,7 +236,7 @@
    FreeFOAM will try to load the library specified by it.
 2. If the sub-dictionary +PstreamImplementation+ exists in the global
    'controlDict' file (see above), it reads the value of the entry
-   +PstreamImplementation::configName+ therein. This entry can be overriden by
+   +PstreamImplementation::configName+ therein. This entry can be overridden by
    setting the +FREEFOAM_PSTREAM_CONFIG+ environment variable. It then expects
    that a sub-dictionary of +PstreamImplementation+ with the name specified in
    +FREEFOAM_PSTREAM_CONFIG+ or +PstreamImplementation::configName+ exists. If
--- a/src/OpenFOAM/meshes/polyMesh/polyBoundaryMesh/polyBoundaryMesh.C
+++ b/src/OpenFOAM/meshes/polyMesh/polyBoundaryMesh/polyBoundaryMesh.C
@@ -471,7 +471,7 @@
                     Pout<< " ***Problem with boundary patch " << patchI
                         << " named " << bm[patchI].name()
                         << " of type " <<  bm[patchI].type()
-                        << ". The patch seems to be preceeded by processor"
+                        << ". The patch seems to be preceded by processor"
                         << " patches. This is can give problems."
                         << endl;
                 }
